src/app-layout/visual-refresh-toolbar/skeleton/styles.scss

```diff
@@ -76,10 +76,11 @@
   grid-row: 1 / -1;
   grid-column: 1 / -1;
   background: awsui.$color-background-container-content;
-  z-index: 850;
+  z-index: constants.$drawer-z-index;
   opacity: 1;
   @include mobile-only {
     inline-size: 100%;
+    z-index: constants.$drawer-z-index-mobile;
```

Comment:
Are these z-indexes really needed? There is another declaration `src/app-layout/visual-refresh-toolbar/navigation/styles.scss` which overrides the current one

Reply:
Yeah we need this here because it is responsible for the grid layout. Otherwise drawers won't be displayed above content on mobile.

Reply:
Got it. Maybe the desktop one on the line 79 is not needed then, but also fine to keep it